Meta/Lagom/Tools/CodeGenerators/LibWeb/GenerateCSSPropertyID.cpp

```diff
@@ -361,6 +362,8 @@ Optional<PropertyID> property_id_from_camel_case_string(StringView string)
 
 Optional<PropertyID> property_id_from_string(StringView string)
 {
+    if (Infra::is_ascii_case_insensitive_match(string, "all"sv))
+        return PropertyID::All;
```

Comment:
I think this code needs to also be output in `property_id_from_camel_case_string()` above.